docs/_articles/en/release-information.md

```diff
@@ -3,7 +3,29 @@ title: Release Information
 lang: en
 ---
 
-### To update the plugin, please follow the linked  [instructions](./en/getting-started/install/#upgrade-plug-in).
+### To update the plugin, please follow the linked [instructions](./en/getting-started/install/#upgrade-plug-in)
+
+## [v2.12.0](https://github.com/forcedotcom/sfdx-scanner/tree/v2.12.0) (09-29-2021)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.11.0...v2.12.0)
+
+### Release Summary
+
+* Updated PMD to 6.38.0 and RetireJS to 2.2.5
+
+**Merged pull requests:**
+
+- @W-9930749@: Reverting the heartbeat scheduling and timeout to the de… [\#546](https://github.com/forcedotcom/sfdx-scanner/pull/546)
+- @W-9930749@: Fixing error in dedup key. [\#545](https://github.com/forcedotcom/sfdx-scanner/pull/545)
+- @W-9930749@: Added dedup key to Github action. [\#544](https://github.com/forcedotcom/sfdx-scanner/pull/544)
+- @W-9930749@: Cancelled, i.e. timed-out, heartbeat should now cause pa… [\#543](https://github.com/forcedotcom/sfdx-scanner/pull/543)
+- @W-9930749@: Added a 60-minute timeout to the heartbeat action. [\#542](https://github.com/forcedotcom/sfdx-scanner/pull/542)
+- @W-9930749@: Added a 60-minute timeout to the heartbeat action. [\#540](https://github.com/forcedotcom/sfdx-scanner/pull/540)
```

Comment:
These are all relating to the heartbeat, and probably don't need to be in our release notes.

Reply:
We decided a couple of releases ago that we'll account for all the PRs here. The `Release Summary` above would list the changes that a user may care about.

Reply:
Yes, I agree @rmohan20  that way we can be consistent.